wikipedia_cleanup/predictor.py

```diff
@@ -194,3 +203,30 @@ def _save_cache(self, possible_cached_mapping: Path) -> None:
 
     def _adjust_cache(self) -> None:
         pass
+
+
+class MeanPredictor(StaticPredictor, RegressionPredictor):
```

Comment:
Git did not recognize that you changed the position of the mean predictor in this file. You did not change any logic here right? (For now I did not check this.)

Reply:
Same for the LastChangePredictor

Reply:
Alright. That should not introduce any bugs.